common/meta/types/src/user_grant.rs

```diff
@@ -26,8 +27,26 @@ pub enum GrantObject {
     Table(String, String),
 }
 
+impl GrantObject {
+    pub fn allow_privilege(&self, privilege: UserPrivilegeType) -> bool {
```

Comment:
Could you give some comments here to explain why we pass this check?

Reply:
sure, thank you for reminding